FoodPin/FoodPin/View/RestaurantDetailHeaderView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  RestaurantDetailHeaderView.swift
+//  FoodPin
+//
+//  Created by Andreea Sburlea on 28.04.2022.
+//
+
+import UIKit
+
+class RestaurantDetailHeaderView: UIView {
+
+    @IBOutlet var headerImageView: UIImageView!
+    @IBOutlet var heartButton: UIButton!
+    @IBOutlet var nameLabel: UILabel! {
+        didSet {
+            nameLabel.numberOfLines = 0
+
+            if let customFont = UIFont(name: "Nunito-Bold", size: 40.0) {
+                nameLabel.font = UIFontMetrics(forTextStyle: .title1).scaledFont(for: customFont)
+            }
+        }
+    }
+    @IBOutlet var typeLabel: UILabel! {
+        didSet {
+            if let customFont = UIFont(name: "Nunito-Bold", size: 20.0) {
+                typeLabel.font = UIFontMetrics(forTextStyle: .headline).scaledFont(for: customFont)
+            }
+        }
+    }
```

Comment:
IBOutlets should be private and the numberOfLines and font can also be set directly from storyboard

Reply:
I know about the numberOfLines and the font but in the book they teach us that can be done in both ways. Should I delete the lines of code and do it in the other way or to leave it like this?